### DISCUSSION TITLE
How to run without docker?

### DISCUSSION THREAD

**Discussion by NexRX** [OpenAS2/OpenAs2App#342] [Q&A] [answered]
HI, I can't run docker on my VPS, do you have any guides to using without docker?

**Comment by uhurusurfa** [comment, marked as answer]
If you use the download package here:
[https://sourceforge.net/projects/openas2/](https://sourceforge.net/projects/openas2/)

...there is a complete set of documentation in the OpenAS2HowTo.pdf that comes with the downloaded package.

**Reply by NexRX** [reply]
Thanks for the help! :)
